frontend/src/components/Header.js

```diff
@@ -22,12 +22,14 @@ const Wrapper = styled.header`
   width: 100%;
   padding: var(--size-m);
 
-  text-align: center;
+  justify-items: center;
+  align-items: center;
```

Comment:
Try a little bit of ordering for css, first display then the display related rules